### ISSUE TITLE
typedef redefinition with different types (DataFormats/L1GlobalTrigger vs DataFormats/L1TGlobal)

### ISSUE THREAD

**Issue by davidlt** [cms-sw/cmssw#14564]
Cling (i.e. Clang) throws errors while comparing two EDM files for RECO:

```
In file included from DataFormatsL1GlobalTrigger_xr dictionary payload:45:
In file included from /cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-05-15-0000/src/DataFormats/L1GlobalTrigger/interface/L1GlobalTriggerObjectMapRecord.h:23:
/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-05-15-0000/src/DataFormats/L1GlobalTrigger/interface/L1GlobalTriggerObjectMapFwd.h:34:33: error: typedef redefinition with different types ('vector<L1GtObject>' vs 'vector<l1t::GlobalObject>')
typedef std::vector<L1GtObject> ObjectTypeInCond;
                                ^
/cvmfs/cms-ib.cern.ch/week1/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-05-15-0000/src/DataFormats/L1TGlobal/interface/GlobalObjectMapFwd.h:34:40: note: previous definition is here
typedef std::vector<l1t::GlobalObject> ObjectTypeInCond;
                                       ^
Error in <TInterpreter::AutoParse>: Error parsing payload code for class L1GlobalTriggerReadoutRecord with content:
```

Apparently there is a typedef (same name), but defined with different type in dictionaries.

/cc @slava77

**Comment by davidlt** [comment]
assign l1

**Comment by slava77** [comment]
@davidlt 
thanks for opening this.
I noticed it about a week ago, but haven't followed up.

**Comment by kpedro88** [comment]
I'm still getting these messages in recent IBs...

**Comment by kpedro88** [comment]
..and the bot apparently decided this was too old and removed the l1 label, can someone reassign it? @slava77 @davidlt

**Comment by Dr15Jones** [comment]
assign l1

**Comment by slava77** [comment]
I think that this was fixed a while back with #18442
